vtm-desktop/src/org/oscim/awt/AwtPaint.java

```diff
@@ -169,9 +185,10 @@ public void setTextSize(float textSize) {
 
     @Override
     public void setTypeface(FontFamily fontFamily, FontStyle fontStyle) {
-        this.fontName = getFontName(fontFamily);
+        Map<Attribute, Object> tmpAttributes = new HashMap<>(this.TEXT_ATTRIBUTES);
```

Comment:
Better avoid here the object creation for each rule or label, can think a way to somehow improve that.

Reply:
Moved it in one method, as methods cannot return two objects, and a new class would cause new instances. Thought of an final static instance, too, but has to be synchronized and reset and refilled. So this is the better alternative I think. If you have improvements, can edit of course :)

Reply:
Thanks, that seems better!